embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -39,6 +40,27 @@ internal sealed class SchemaType(
         override val attrs = mapOf("view.name" to viewName)
     }
 
+    /**
+     * Represents a tap breadcrumb event.
+     * @param viewName The name of the view that the tap event occurred in.
+     * @param type The type of tap event. "tap"/"long_press". "tap" is the default.
+     * @param coords The coordinates of the tap event.
+     */
+    internal class TapBreadcrumb(
```

Comment:
We're trying to not use "breadcrumb" for anything other than a custom breadcrumb, which will just be called breadcrumb. Maybe just call this `Tap`?

Reply:
I'll change the rest to conform to the new name so don't worry about those haha